Testing/SystemTests/tests/analysis/VesuvioTesting.py

```diff
@@ -0,0 +1,514 @@
+from mantid.simpleapi import *
```

Comment:
Did you forget to (re-)move VesuvioTesting.py from Framework/PythonInterface...? Or maybe you need it for some other vesuvio algorithms? It wouldn't be nice to have it duplicated.